tc/sgx/trusted_worker_manager/enclave/work_order_data_handler.h

```diff
@@ -31,28 +31,71 @@ namespace tcf {
         public:
             WorkOrderDataHandler() {}
 
-            explicit WorkOrderDataHandler(const WorkOrderData& wo_data, std::string encrypted_session_key) {
+            explicit WorkOrderDataHandler(ByteArray session_key,
+                                          ByteArray session_key_iv) {
+                this->session_key = session_key;
+                this->session_key_iv = session_key_iv;
+            }
+
+            // Used when input request doesn't have OutData
+            explicit WorkOrderDataHandler(const WorkOrderData& wo_data,
+                                          ByteArray data_encryption_key,
+                                          ByteArray data_iv,
+                                          std::string enc_data_key_str,
+                                          std::string iv) {
                 workorder_data = wo_data;
-                data_encryption_key = HexEncodedStringToByteArray(encrypted_session_key);
+                this->data_encryption_key = data_encryption_key;
+                this->data_iv = data_iv;
+
+                this->enc_data_key_str = enc_data_key_str;
+                this->iv = iv;
```

Comment:
Can you add a comment in code on what is data_encryption_key  and enc_data_key_str

Reply:
done